docs/amber_script.md

```diff
@@ -229,26 +231,56 @@ value for `START_IDX` is 0. The default value for `COUNT` is the item count of
 vertex buffer minus the `START_IDX`.
 
 ```
+# Run the given |pipeline_name| which must be a `compute` pipeline. The
+# pipeline will be run with the given workgroup |x|, |y|, |z| dimensions.
+# Each of the x, y and z values must be a uint32.
 RUN <pipeline_name> <x> <y> <z>
 
+# Run the given |pipeline_name| which must be a `graphics` pipeline. The
+# rectangle at |x|, |y|, |width|x|height| will be rendered.
 RUN <pipeline_name> \
   DRAW_RECT POS <x_in_pixels> <y_in_pixels> \
   SIZE <width_in_pixels> <height_in_pixels>
 
+# Run the |pipeline_name| which must be a `graphics` pipeline. The vertex
+# data must be attached to the pipeline. A start index of 0 will be used
+# and a count of the number of elements in the vertex buffer.
 RUN <pipeline_name> DRAW_ARRAY AS <topology>
-RUN <pipeline_name> DRAW_ARRAY AS <topology> START_IDX <value>
-RUN <pipeline_name> DRAW_ARRAY AS <topology> START_IDX <value> COUNT <value>
-
+# Run the |pipeline_name| which must be a `graphics` pipeline. The vertex
+# data must be attached to the pipeline. A start index of |value| will be used
+# and a count of the number of elements in the vertex buffer.
```

Comment:
count of the number of _remaining_ elements of the vertex buffer.

or use the phrasing similar in to what you have in the indexed case.